crates/supervisor/core/src/config/rollup_config_set.rs

```diff
@@ -0,0 +1,90 @@
+use alloy_primitives::U64;
+use kona_supervisor_types::BlockSeal;
+use std::collections::HashMap;
+
+/// Genesis provides the genesis information relevant for Interop.
+#[derive(Debug, Clone)]
+pub struct Genesis {
+    /// The L1 [`BlockSeal`] that the rollup starts after.
+    pub l1: BlockSeal,
+    /// The L2 [`BlockSeal`] that the rollup starts from.
+    pub l2: BlockSeal,
+}
+
+impl Genesis {
+    /// Creates a new Genesis with the given L1 and L2 block seals.
+    pub const fn new(l1: BlockSeal, l2: BlockSeal) -> Self {
+        Self { l1, l2 }
+    }
+
+    /// Creates a new Genesis from a RollupConfig.
+    pub fn new_from_rollup_genesis(genesis: kona_genesis::ChainGenesis, l1_time: u64) -> Self {
```

Comment:
```suggestion
    pub fn new_from_rollup_genesis(genesis: ChainGenesis, l1_time: u64) -> Self {
```
pls import in module prelude